apps/api/lib/validations/user.ts

```diff
@@ -4,6 +4,7 @@ import { checkUsername } from "@calcom/lib/server/checkUsername";
 import { _UserModel as User } from "@calcom/prisma/zod";
 import { iso8601 } from "@calcom/prisma/zod-utils";
 
+import { isValidBase64Image } from "~/lib/utils/isValidBase64";
```

Comment:
missed this during review, this file is now named `sValidBase64Image.ts` not `isValidBase64.ts`